File: src/main/java/com/xuancong/employee_management/dto/address/AddressResponse.java
package com.xuancong.employee_management.dto.address;

import com.xuancong.employee_management.model.Address;

public record AddressResponse(
        Long id,
        Long countryId,
        String countryName,
        Long provinceId,
        String provinceName,
        Long districtId,
        String districtName,
        String specificAddress
) {

}

File: frontend/models/address/addressResponse.ts
export type AddressResponse = {
  id: number;
  countryId: number;
  countryName: string;
  provinceId: number;
  provinceName: string;
  districtId: number;
  districtName: string;
  specificAddress: string;
};




export type AddressCreateRequest = {
  countryId: number;
  provinceId: number;
  districtId: number;
  specificAddress: string;
};